uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/GoogleSettings.java

```diff
@@ -221,6 +226,18 @@ public void forwardDeviceTime(long seconds) {
             // Confirm setting date
             final UiObject okBtn = UiAutomatorUtils.obtainUiObjectWithText("OK");
             okBtn.click();
+
+            // Reset to Automatic Time Zone to avoid chain validation failures
+            // Some cases fail if you do this, however, so setting it to a boolean parameter
+            if (enableTimeZone) {
+                try {
+                    Thread.sleep(TimeUnit.SECONDS.toMillis(3));
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+
+                AdbShellUtils.enableAutomaticTimeZone();
```

Comment:
but this will now reset the time, so how is this still useful? Does it still have an effect on password reset test cases?

Reply:
From what I saw, it still expires the token. I wanted to double check this with the team, but this is helping eliminate chain validation errors. Also, password reset cases are actually not being ran in the pipeline at the moment, they were unreliable last time I tried them and are pretty time consuming, so they extend the run time on firebase.

Reply:
Looks like this is not the case actually, the token is not being expired.